python/docs/events.md

```diff
@@ -740,7 +753,7 @@ Content for:
   ```
   {
       "workflowStatus": {... "workflow status object" ...},
-      "moveToSlug": null | "closed"
+      "targetStatus": null | "closed"
   }
```

Comment:
I know that you only change the documentation about that, but I have searched "moveToSlug" in the code and there are some in frontend code. Maybe we can talk with them about that to avoid bugs

Reply:
I was just trying to update the documentation to reflect the current behaviour. I'll repeat the event again in case I did it wrongly. I'll post my results.

Reply:
I'll reproduce the event launched, as I was just trying to update the documentation to reflect the real behaviour. I'll post the results

Reply:
Ok, but maybe back works but front not because they have some moveToSlug in their front code.

Reply:
Both in my branch and in my branch reply with a targetStatus.

```
INFO:      [EV: manager.publish]: Publish to 'projects.ItoWRXMJEe6DzzcylxXcsQ': '{"type": "workflowstatuses.delete", "content": {"workflowStatus": {"id": "yye0Q3MrEe6DzzcylxXcsQ", "name": "New", "color": 1, "order": 104, "workflow": {"id": "ItoWSHMJEe6DzzcylxXcsQ", "name": "Main", "slug": "main"}}, "targetStatus": {"id": "ItoWS3MJEe6DzzcylxXcsQ", "name": "In progress", "color": 3, "order": 3, "workflow": {"id": "ItoWSHMJEe6DzzcylxXcsQ", "name": "Main", "slug": "main"}}}, "correlationId": "49123e4c1a17448ba1958e590be730e1"}'
```

Another option is to remove this change from the documentation and manage it parallelly.

Reply:
I've undo this change. We can comment this with front-end and fix it in another PR.